README.md

```diff
@@ -2,14 +2,17 @@
 **Distributed Public Key Infrastructure which runs on HoloChain**
 * The frame for the application is created.
 * Created the User.
```

Comment:
code cleanup:  lets get a consistent capitalization scheme.  in the users zome the file names are: "User.json" and "users.js"  and in other zomes it's different.  Lets pick one (probably lower case) and use that the same way everywhere.  I think best is to also use the exact same word, i.e. "users.json" and "users.js" just like you have it for the keyRegistration zome.  And we should also be consistent about plural vs singular.  It may seem like a small thing, but for a person reading the code, if it's not consistent it makes you think that the difference matters in some way.  So its very helpful for there not to be a naming difference across things that are supposed to be the same.